scripts/circle-ci/run-circle-tests.sh

```diff
@@ -40,7 +40,7 @@ case $CIRCLE_NODE_INDEX in
     0) ./gradlew $TEST_CONTAINER_ARGS check $CONTAINER_0_EXCLUDE_ARGS ;;
     1) ./gradlew $TEST_CONTAINER_ARGS ${CONTAINER_1[@]} ;;
     2) ./gradlew $TEST_CONTAINER_ARGS ${CONTAINER_2[@]} -x :atlasdb-ete-tests:longTest ;;
-    3) ./gradlew $TEST_CONTAINER_ARGS ${CONTAINER_3[@]} ;;
+    3) ./gradlew $TEST_CONTAINER_ARGS ${CONTAINER_3[@]} -x :atlasdb-jepsen-tests:jepsenTest ;;
```

Comment:
Might be safer to add a line like this to the check call above as one ' delimited string, this would stop problems arising from people moving the tests around containers.